dashboard/app/controllers/api/v1/section_libraries_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::SectionLibrariesController < Api::V1::JsonApiController
+  check_authorization
+
+  # gets the libraries from all members of all sections I am a part of
+  # GET api/libraries/section/all
+  def get_classmate_published_libraries
+    sections = current_user.sections + current_user.sections_as_student
+    projects = []
```

Comment:
What do you think about re-naming this variable to `libraries`?

Reply:
💯